include/tvm/meta_schedule/database.h

```diff
@@ -357,6 +357,22 @@ class Database : public runtime::ObjectRef {
    */
   TVM_DLL static Database JSONDatabase(String path_workload, String path_tuning_record,
                                        bool allow_missing);
+  /*!
+   * \brief A database composed of multiple databases, allowing users to guide IR rewriting using
+   * combined knowledge of those databases. To each query, it returns the best record among all the
+   * databases given.
+   * \param databases The list of databases to be combined.
+   * \return The combined database.
+   */
+  TVM_DLL static Database UnionDatabase(Array<Database, void> databases);
```

Comment:
When I saw `UnionDatabase` and `OrderedUnionDatabase`, I initially thought the opposite: I thought `OrderedUnionDatabase` sorts tuning records and returns the best one lol 
This might be because I usually assign the name without prefix "Ordered/Sorted" by considering "unsorted" as default. I guess this is a matter of preference, so wonder how other folks think.

Reply:
I think given `Union Database` it's easy to think of a database which returns the best result among all databases given, and this is consistent with the database api of get top k. For `OrderedUnionDatabase` I agree the name is a bit not straghtforward, maybe it could be changed to something like `FirstAvailableUnionDatabase`.

Reply:
😂 yeah this is a valid perspective too...Is there any better naming for them?

Reply:
Thank you @sunggg!!! It definitely clarifies a lot! adding this to the doc